doc/dox_comments/header_files/asn_public.h

```diff
@@ -21,6 +21,61 @@
 */
 int wc_InitCert(Cert*);
 
+/*!
+     \ingroup ASN
+
+     \brief This function allocates a new cert structure for use during
+     cert operations without the application having to allocate the structure
+     itself. The cert structure is also initialized by this function removing
+     the need to call wc_InitCert(). When the application is finish using the
+     allocated cert structure wc_CertFree() must be called.
+
+     \return A pointer to the allocated cert structure.
```

Comment:
mention it can return null on memory allocation failure.

Reply:
Good idea. Updated.